salt/utils/templates.py

```diff
@@ -345,20 +346,12 @@ def render_jinja_tmpl(tmplstr, context, tmplpath=None):
         jinja_env = jinja2.Environment(undefined=jinja2.StrictUndefined,
                                        **env_args)
 
-    jinja_env.filters['strftime'] = salt.utils.date_format
-    jinja_env.filters['sequence'] = salt.utils.jinja.ensure_sequence_filter
-    jinja_env.filters['yaml_dquote'] = salt.utils.yamlencoding.yaml_dquote
-    jinja_env.filters['yaml_squote'] = salt.utils.yamlencoding.yaml_squote
-    jinja_env.filters['yaml_encode'] = salt.utils.yamlencoding.yaml_encode
-    jinja_env.filters['is_ip'] = salt.utils.network.is_ip_filter  # check if valid IP address
-    jinja_env.filters['is_ipv4'] = salt.utils.network.is_ipv4_filter  # check if valid IPv4 address
-    jinja_env.filters['is_ipv6'] = salt.utils.network.is_ipv6_filter  # check if valid IPv6 address
-    jinja_env.filters['ipaddr'] = salt.utils.network.ipaddr  # filter IP addresses
-    jinja_env.filters['ipv4'] = salt.utils.network.ipv4  # filter IPv4-only addresses
-    jinja_env.filters['ipv6'] = salt.utils.network.ipv6  # filter IPv6-only addresses
-    jinja_env.filters['ip_host'] = salt.utils.network.ip_host  # return the network interface IP
-    jinja_env.filters['network_hosts'] = salt.utils.network.network_hosts  # return the hosts within a network
-    jinja_env.filters['network_size'] = salt.utils.network.network_size  # return the network size
+    for function_name in context['salt'].keys():
+        # Simply iterating over the keys forces the execution modules to load
+        # and process the jinja env decorator(s)
+        pass
```

Comment:
I rebased on latest develop, so the prior comment on this loop is hidden. I'd still like to get feedback from someone on the salt team more familiar with the loader, regarding this loop and whether there is an easier/better way to get the execution modules to load and process the decorator.